fog/ledger/enclave/impl/Cargo.toml

```diff
@@ -17,6 +17,16 @@ mc-common = { path = "../../../../mobilecoin/common", default-features = false }
 mc-crypto-keys = { path = "../../../../mobilecoin/crypto/keys", default-features = false }
 mc-util-serial = { path = "../../../../mobilecoin/util/serial" }
 mc-sgx-report-cache-api = { path = "../../../../mobilecoin/sgx/report-cache/api" }
+mc-crypto-rand = { path = "../../../../mobilecoin/crypto/rand" }
+mc-util-grpc = { path = "../../../../mobilecoin/util/grpc" }
+mc-transaction-core = { path = "../../../../mobilecoin/transaction/core" }
+mc-sgx-compat = { path = "../../../../mobilecoin/sgx/compat", default-features = false }
+byteorder = "1"
```

Comment:
Please don't use byteorder crate, it is deprecated and all the functionality is in the core language now

Reply:
done